### ISSUE TITLE
Events DayPicker / search clips itself on rotation from landscape to portrait

### ISSUE THREAD

**Issue by OnlyInAmerica** [Burning-Man-Earth/iBurn-iOS#41]
If you transition the Events view from landscape to portrait, the day picker and search view do not adjust their bounds to the reduced portrait width.

![Event rotation bug](http://i.imgur.com/nLvtspg.png)

**Comment by chrisballinger** [comment]
Yeah there's a lot of bugs associated with that whole thing. I'm gonna see if I can track down another iOS developer to take a look.

**Comment by muloka** [comment]
In addition to this it seems that in landscape the search bar remains the same size. 

![Image](http://i.imgur.com/P0RJgmF.jpg)

**Comment by chrisballinger** [comment]
This should be fixed by the latest pull request!
